pkg/systemd/logs.html

```diff
@@ -29,95 +29,100 @@
+                      <span translate>Search the logs with a combination of terms:</span>
+                      <ul>
+                          <li><span translate>qualifiers</span>, <span translate>e.g.</span> 'priority:', 'identifier:', 'service:'</li>
+                          <li><span translate>log fields</span>, <span translate>e.g.</span> '_EXE=/usr/bin/python'</li>
+                          <li translate>any free-form string as regular expression</li>
+                      </ul>
+                      <span class="help-links">
+                          <a href="https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_systems_using_the_rhel_8_web_console/reviewing-logs_system-management-using-the-rhel-8-web-console#filtering-logs-in-the-web-console_reviewing-logs" target="blank" rel="noopener noreferrer"><span translate>Learn more</span><i class="fa fa-external-link fa-xs"></i></a>
+                          <a href="https://www.freedesktop.org/software/systemd/man/journalctl.html" target="blank" rel="noopener noreferrer"><span translate>journalctl manpage</span><i class="fa fa-external-link fa-xs"></i></a>
+                      </span>
+                      <span class="journal-cmd">
+                          <pre id="journal-query"></pre>
+                          <button class="copy-cmd" id="journal-cmd-copy"><i class="fa fa-clipboard fa-xs"></i></button>
+                      </span>
+                    </div>
                   </div>
-                </div>
-            </span>
+              </span>
+          </div>
         </div>
-      </div>
 
-      <div class="filter-separator"></div>
+        <div class="filter-separator"></div>
 
-      <div class="filter-actions">
+        <div class="filter-actions">
 
-        <span class="filters-toggle" hidden>
-          <button class="link-button" translate>Show filters</button>
-        </span>
+          <span class="filters-toggle" hidden>
+            <button class="link-button" translate>Show filters</button>
+          </span>
 
-        <button id="journal-follow" class="pf-c-button pf-m-secondary" type="button" translate>Pause</button>
-      </div>
-    </div>
-
-    <div id="journal-box" class="container-fluid"></div>
+          <button id="journal-follow" class="pf-c-button pf-m-secondary" type="button" translate>Pause</button>
+        </div>
+      </section>
+      <section id="journal-box" class="pf-c-page__main-section pf-m-light"></section>
+    </main>
   </div>
 
-  <div id="journal-entry" class="container-fluid" hidden>
-    <ol class="breadcrumb">
-      <li><a role="link" tabindex="0" id="journal-navigate-home" translate="yes">Logs</a></li>
-      <li id="journal-entry-crumb" class="active" translate="yes">Entry</li>
-    </ol>
-    <h2 id="journal-entry-heading"></h2>
-    <table class="info-table-ct panel panel-default" id="journal-entry-fields">
-    </table>
+  <div id="journal-entry" class="pf-c-page" hidden>
```

Comment:
Somewhere here is some problem, there is massive gap on top of a log entry.
![massivegap](https://user-images.githubusercontent.com/12330670/95781626-86d78680-0cce-11eb-821b-e2e108e3f2e7.png)

Reply:
Oh and it misses breadcrumb. Can you please also write test for it? Thanks!

Reply:
This is the current master:
![masternow](https://user-images.githubusercontent.com/12330670/95825012-88399b00-0d30-11eb-9ace-c831e1bcb9a4.png)

Reply:
Fixed. Thanks for noticing!
![Screen Shot 2020-10-13 at 11 16 29](https://user-images.githubusercontent.com/14921356/95841505-bd041d00-0d45-11eb-9b2b-6ee5ef866c35.png)

Reply:
Ups I forgot the test - will repost again.